### PR TITLE
Layout documentation

### PR DESCRIPTION
**Pull request by Karan-Bharti**
alert

### CODE REVIEW

File: Alert/alert.html [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+        <a href="" class="drawer-button">Button</a>
+        <a href="" class="drawer-card">Card</a>
+        <a href="" class="drawer-image">Image</a>
+        <a href="" class="drawer-input">Input</a>
+        <a href="" class="drawer-grid">Grid</a>
+        <a href="" class="drawer-snackbar">Snackbar</a>
+        <a href="" class="drawer-lists">Lists</a>
+    </div>
+    <div class="main-section">
+        <h2 style="padding-bottom: 2rem; padding-top: 2rem;">Alert</h2>
```
Comment (krishanu2911) [inline Alert/alert.html]: try not to use inline styling

**Review by krishanu2911 — COMMENTED** [review]
Everything seems good to go just want to suggest that try not to use inline CSS

**Review by sameep-s — APPROVED** [review]
Just look at comments and then you can merge.

File: Alert/alert.html (hunk 1 above)
Comment (Karan-Bharti) [reply Alert/alert.html]: Yes I removed inline styling.

**Pull request merged**